Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -446,27 +446,30 @@ def check_new_version(formula, tap_remote_repo, args:, version: nil, url: nil, t
       return if version.null?
     end
 
-    check_throttle(formula, version)
+    check_formula_throttle(formula, tap_remote_repo)
     check_closed_pull_requests(formula, tap_remote_repo, args: args, version: version)
   end
 
-  def check_throttle(formula, new_version)
-    throttled_rate = formula.tap.audit_exceptions.dig(:throttled_formulae, formula.name)
-    return if throttled_rate.blank?
+  def check_formula_throttle(formula, tap_remote_repo)
+    throttle_days = formula.tap.audit_exceptions.dig(:throttled_formulae, formula.name) || 0
+    return if throttle_days.zero?
 
-    formula_suffix = Version.new(new_version).patch.to_i
-    return if formula_suffix.modulo(throttled_rate).zero?
-
-    odie "#{formula} should only be updated every #{throttled_rate} releases on multiples of #{throttled_rate}"
+    GitHub.check_recently_merged_pull_requests(
```

Comment:
I agree that I'd want to see @homebrew/core express a need for this before it's  implemented here.